html/typing2.js

```diff
@@ -0,0 +1,43 @@
+inserChar = '|'
```

Comment:
can any JAVASCRIPT GURUS show up and help us get this memery working

we want the console text effect on all text in the arcade game